<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Lucida Handwriting&quot;;</string>
    </property>
    <property name="text">
     <string>    WELCOME</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Candara&quot;;</string>
    </property>
    <property name="text">
     <string>AeroBus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(67, 66, 66);
	color: rgb(255, 255, 255);
	
}
QPushbutton:enabled{
background-color: rgb(85, 0, 255);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
background-color: rgb(13, 47, 114);
	color: rgb(255, 255, 254);
}
QPushButton:hover:!pressed{
background-color: rgb(189, 110, 12);
	color: rgb(12, 47, 112);
}
QPushButton:disabled{
background-color: rgb(170, 170, 170);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>T815</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(67, 66, 66);
	color: rgb(255, 255, 255);
}
QPushbutton:enabled{
background-color: rgb(85, 0, 255);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
background-color: rgb(13, 47, 114);
	color: rgb(255, 255, 254);
}
QPushButton:hover:!pressed{
background-color: rgb(189, 110, 12);
	color: rgb(12, 47, 112);
}
QPushButton:disabled{
background-color: rgb(170, 170, 170);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>T789</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(67, 66, 66);
	color: rgb(255, 255, 255);
}
QPushbutton:enabled{
background-color: rgb(85, 0, 255);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
background-color: rgb(13, 47, 114);
	color: rgb(255, 255, 254);
}
QPushButton:hover:!pressed{
background-color: rgb(189, 110, 12);
	color: rgb(12, 47, 112);
}
QPushButton:disabled{
background-color: rgb(170, 170, 170);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>T870</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Please Choose Your Bus</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/nurl(:/newPrefix/clipart2228037.png)ewPrefix/clipart2228037.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/newPrefix/clipart2228037.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
